shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -1128,10 +1170,13 @@ @interface FlutterTextInputPlugin ()
 @property(nonatomic, readonly)
     NSMutableDictionary<NSString*, FlutterTextInputView*>* autofillContext;
 @property(nonatomic, assign) FlutterTextInputView* activeView;
+@property(nonatomic, retain) FlutterTextInputViewAccessibilityHider* inputHider;
 @end
 
 @implementation FlutterTextInputPlugin
 
+NSTimer* _enableFlutterTextInputViewAccessibilityTimer;
+
```

Comment:
This is a global variable as written, you need to put this in braces.

Reply:
oops nice catch. will update